src/HDFWriterModule.h

```diff
@@ -9,16 +9,16 @@
 
 #pragma once
 
+#include "FlatbufferMessage.h"
```

Comment:
You don't like forward declarations? They can drastically decrease compile times...

Reply:
It's more about not actively searching for places where they can be used. I can add one in this case but if you know some tool that automates finding opportunities for forward declarations, that would be great.

Reply:
I asked because it looks like you actively removed one here.
When writing new stuff I think it is just a case of being aware of the cost of adding includes to headers: forward declare by default and only `#include` if that doesn't work.

Reply:
I can't be bother to figure out exactly why it happened but seeing as you added this forward declarations to this file very recently, it seems likely that the reason for it disappearing was because I branched off of master before that change had been merged.

Reply:
Considering the changes I had to do to make the forward declaration work, my previous comment seems like a very likely reason.